packages/shared/routes/dashboard/governance/views/GovernanceEventDetails.svelte

```diff
@@ -98,18 +126,93 @@
-        <Icon icon="info" classes="ml-auto mt-0" />
-    </DashboardPane>
+    <div class="row-span-1">
+        <DashboardPane classes="w-full h-full flex flex-row flex-shrink-0 overflow-hidden p-6">
+            <div class="space-y-5">
+                <div>
+                    <Text type="p" smaller classes="mb-3 text-gray-700" overrideColor
+                        >{localize('views.governance.votingPower.title')}</Text
+                    >
+                    <Text type="h2" classes="inline-flex items-end">{account?.balance}</Text>
+                </div>
+                {#if event?.status?.status === ParticipationEventState.Upcoming}
+                    <div>
+                        <Text type="p" smaller classes="mb-3 text-gray-700" overrideColor
+                            >{localize('views.governance.eventDetails.votingOpens')}</Text
+                        >
+                        <Text type="h3" classes="inline-flex items-end"
+                            >{milestoneToDate(event?.information?.milestoneIndexCommence).toString()}</Text
+                        >
+                    </div>
+                {/if}
+                {#if event?.status?.status === ParticipationEventState.Upcoming || event?.status?.status === ParticipationEventState.Commencing}
+                    <div>
+                        <Text type="p" smaller classes="mb-3 text-gray-700" overrideColor
+                            >{localize('views.governance.eventDetails.countingStarts')}</Text
+                        >
+                        <Text type="h3" classes="inline-flex items-end"
+                            >{milestoneToDate(event?.information?.milestoneIndexStart).toString()}</Text
+                        >
+                    </div>
+                {/if}
+                {#if event?.status?.status === ParticipationEventState.Commencing}
+                    <div>
+                        <Text type="p" smaller classes="mb-3 text-gray-700" overrideColor
+                            >{localize('views.governance.eventDetails.countingLength')}</Text
+                        >
+                        <Text type="h3" classes="inline-flex items-end">{getBestTimeDuration(length)}</Text>
+                    </div>
+                {/if}
+                {#if event?.status?.status === ParticipationEventState.Holding || event?.status?.status === ParticipationEventState.Ended}
+                    <div>
+                        <Text type="p" smaller classes="mb-3 text-gray-700" overrideColor
+                            >{localize('views.governance.eventDetails.votesCounted')}</Text
+                        >
+                        <Text type="h3" classes="inline-flex items-end">{delineateNumber(totalVotes.toString())}</Text>
+                    </div>
+                {/if}
+                {#if event?.status?.status === ParticipationEventState.Holding || event?.status?.status === ParticipationEventState.Ended}
+                    <div>
```

Comment:
You have a duplicate if statement. It would be better to move this to the if statement right above.